### ISSUE TITLE
Use community QtWebKit >= 5.9

### ISSUE THREAD

**Issue by ariya** [ariya/phantomjs#15342]
To support modern web standards (including but not limited to ES 2015 and beyond), future version of PhantomJS must require an updated version of QtWebKit, the one developed by @annulen at https://github.com/annulen/webkit.

Among others, the source packages for this flavor of QtWebKit is available at e.g. http://download.qt.io/community_releases/5.9/5.9.0-final/.

**Comment by ariya** [comment]
However, many Linux distributions do not package this community QtWebKit yet.

There is some hope, for instance Debian 9.3 has version 5.7.1:

```
$ apt-cache show libqt5webkit5
Package: libqt5webkit5
Source: qtwebkit-opensource-src
Version: 5.7.1+dfsg-1
Installed-Size: 38060
Maintainer: Debian Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Architecture: amd64
```

We of course want an updated version (to get ES 2015 etc, see [the comparison](https://github.com/annulen/webkit/wiki/Comparison-with-QtWebKit-5.6)).

Checking Debian sid (unstable), it seems that an updated package is available: [qtwebkit-opensource-src 5.212.0](https://packages.debian.org/source/sid/qtwebkit-opensource-src). Hopefully, it will make into the next stable of Debian, as well as Ubuntu.

**Comment by ariya** [comment]
This has been implemented.
